includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1401,6 +1409,11 @@ public static function print_admin_notice() {
 			);
 		}
 
+		// Add notice for PHP fatal error during validation request.
+		if ( 'post' === get_current_screen()->base && self::POST_TYPE_SLUG === get_current_screen()->post_type && get_post() ) {
+			self::render_php_fatal_error_admin_notice( get_post() );
+		}
```

Comment:
`get_post()` is being called twice which is bad for performance and could potentially cause weird bugs where the second call returns something else than the first call.
```suggestion
		// Add notice for PHP fatal error during validation request.
		$post = get_post();
		if ( $post instanceof WP_Post && 'post' === get_current_screen()->base && self::POST_TYPE_SLUG === get_current_screen()->post_type ) {
			self::render_php_fatal_error_admin_notice( $post );
		}
```